tests/fake_app.py

```diff
@@ -0,0 +1,20 @@
+from flask import Flask
+
+from flasql.views import GraphQLView
+
+
+def create_graphql_view(**kwargs):
+    schema = kwargs.pop('schema')
+
+    kwargs.update({
+        'schema': schema,
+        'enable_graphiql': True,
+    })
+
+    return GraphQLView.as_view('graphql', **kwargs)
+
+
+def create_app(**kwargs):
+    app = Flask(__name__)
+    app.add_url_rule('/graphql', view_func=create_graphql_view())
+    return app
```

Comment:
The `app` is used to deliver a context to the tests.

It's not actively used *right now* but I would like to add more tests to actually test that the graphql view is functioning properly.